Merge Conflict/Assets/Scripts/TagHandler.cs

```diff
@@ -0,0 +1,30 @@
+/**********************************************************************************************************************
+Name:          TagHandler
+Description:   Contains all tags, which are used to identify objects. With CompareTags() its possible to compare the tag 
+               from the given GameObject with a tag from the PossibleTags "list".
+Author(s):     Markus Haubold
+Date:          2024-03-04
+Version:       V1.0
+TODO:          - ongoing: add new tags
+**********************************************************************************************************************/
+
+using UnityEngine;
+
+//add all tags which are used in unity!
+public static class TagHandler
```

Comment:
rename to simply "Tags" for simplicity

Reply:
According to this [guideline](https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md#c-coding-standards-and-naming-conventions) for naming classes i will rename it with Tag.